README.md

```diff
@@ -63,3 +63,4 @@ For all pull requests, please make sure there's an issue created for it first an
 - Czech: [Michal Moudrý](https://github.com/MichalMoudry)
 - Spanish: [Leonardo González Castro](https://github.com/OnlyOnePro), [Breece W](https://github.com/BreeceW)
 - Hungarian: [Kristóf Kékesi](https://github.com/KristofKekesi)
+- Portuguese [BR]: [Vinicius Rodrigues](https://github.com/Suburbanno)
```

Comment:
thanks for adding this! I'm going to ask all translators to add their names from now on 👍🏽